library/src/main/java/com/chuckerteam/chucker/api/internal/support/FormatUtils.java

```diff
@@ -73,7 +59,11 @@ public static String formatJson(String json) {
 
     public static String formatXml(String xml) {
         try {
-            Transformer serializer = SAXTransformerFactory.newInstance().newTransformer();
+            SAXTransformerFactory transformerFactory = (SAXTransformerFactory) SAXTransformerFactory.newInstance();
+            transformerFactory.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
+            transformerFactory.setAttribute("http://javax.xml.XMLConstants/property/accessExternalDTD", "");
+            transformerFactory.setAttribute("http://javax.xml.XMLConstants/property/accessExternalStylesheet", "");
```

Comment:
I'm not sure to understand this 2 lines. It's like we are allowing the usage of external DTDs and Stylesheets. If I'm right, the security is still there, the transformer will download the external resources.